lib/CFRPageActions.jsm

```diff
@@ -914,6 +931,18 @@ const CFRPageActions = {
     this.PageActionMap = PageActionMap;
     this.RecommendationMap = RecommendationMap;
   },
+
+  /**
+   * Reload the l10n Fluent files for all PageActions
+   */
+  reloadL10n() {
+    for (const win of Services.wm.getEnumerator("navigator:browser")) {
```

Comment:
We were using EveryWindow elsewhere which seems to be the new way to enumerate browser windows?

Reply:
OK, I will swap that in.

Reply:
Hmm, looks like there are a few other modules that are still calling `getEnumerator`, wonder if we should have a separate patch to refactor it?

Reply:
Ah for `clearRecommendations` sure those can be done together separately